client/src/components/alerts/component.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { CheckCircleIcon, XCircleIcon, ExclamationIcon } from '@heroicons/react/solid';
+
+import { ALERT_CLASSES } from './constants';
+import { AlertsProps } from './types';
+
+const Alerts: React.FC<AlertsProps> = ({ className, items }: AlertsProps) => {
+  if (!items) return null;
+  if (Array.isArray(items) && !items.length) return null;
+
+  const alerts = Array.isArray(items) ? items : [items];
+
+  const getIcon = (type) => {
```

Comment:
I'm fairly sure there's an easier way of doing this, but for the life of me I can't remember how.

Reply:
I don't like so much having props in different formats `AlertItemsProp` `AlertItemsProp[]`. To me should be always an array, and you can choice inside the component depending of the length